### PR TITLE
Include partitioned tables, hide partitions

### PR DESCRIPTION
**Pull request by jonels-msft**
This continues the work of https://github.com/microsoft/pgtoolsservice/pull/422, covering more areas of the UI.

* Hides partitions in the database "Manage" screen (fixes https://github.com/microsoft/azuredatastudio-postgresql/issues/126)
* Includes partitioned tables and their columns in query completion popups, and excludes individual partitions.

### CODE REVIEW

**Review by nasc17 — APPROVED** [review]
Could you include screenshots of fix

**Comment by jonels-msft** [comment]
Sure, I'll add screenshots.

File: ossdbtoolsservice/metadata/metadata_service.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 UNION
 SELECT schemaname AS schema_name, tablename AS object_name, 't' as type FROM pg_tables
     WHERE schemaname NOT ILIKE 'pg_%' AND schemaname != 'information_schema'
+      AND tablename NOT IN ( SELECT relname FROM pg_class WHERE relispartition )
```
Comment (jonels-msft) [inline ossdbtoolsservice/metadata/metadata_service.py:25]: I worried this change might introduce inefficiency, but I verified it generates the same EXPLAIN plan as using a CTE for the subquery.

**Comment by jonels-msft** [comment]
### Before

#### Manage database

before-manage

#### Autocomplete popup

before-completion-tables

### After

#### Manage database

after-manage

#### Autocomplete popup

after-completion-tables

(plus columns in the partitioned table)
after-completion-column

**Review by nasc17 — APPROVED** [review]

**Pull request merged**